src/bloom.h

```diff
@@ -80,6 +80,7 @@ class CBloomFilter
      * nFlags should be one of the BLOOM_UPDATE_* enums (not _MASK)
      */
     CBloomFilter(unsigned int nElements, double nFPRate, unsigned int nTweak, unsigned char nFlagsIn);
+    CBloomFilter(unsigned int nElements, double nFPRate, unsigned int nTweak, unsigned char nFlagsIn, uint32_t nMaxFilterSize);
```

Comment:
why we need two constructor for the rolling bloom filter?

Reply:
you're right, i'll work on this one.